tests/benchmarks/micro_benchmarks/test_ib_traffic_performance.py

```diff
@@ -242,17 +259,25 @@ def test_ib_traffic_performance(self, mock_gpu):
 """
 
         # Check function process_raw_data.
-        # Positive case - valid raw output.
+        # Positive cases - valid raw output.
         os.environ['OMPI_COMM_WORLD_RANK'] = '0'
         assert (benchmark._process_raw_result(0, raw_output_0))
-
         for metric in benchmark.result:
             assert (metric in benchmark.result)
             assert (len(benchmark.result[metric]) == 1)
             assert (isinstance(benchmark.result[metric][0], numbers.Number))
-        # Negative case - valid raw output.
-        assert (benchmark._process_raw_result(0, raw_output_1) is False)
+        values = list(benchmark.result.values())[1:]
+        assert(all(value == [-1.0] for value in values))
```

Comment:
need a space after `assert`, you can run `python3 setup.py lint` locally to check